packages/code-du-travail-frontend/src/common/RelatedItems.js

```diff
@@ -102,11 +102,13 @@ export const RelatedItems = ({ items = [] }) => {
                     <Link
                       href={`/${route}${rootSlug ? "/[slug]" : ""}`}
                       as={`/${route}${rootSlug}${hash}`}
-                      onClick={matopush([
-                        "trackEvent",
-                        "selectRelated",
-                        `${route}${rootSlug}${hash}`
-                      ])}
+                      onClick={() =>
```

Comment:
I wonder if we should we add  onCLick handler on `<Link/>` or on its child `<a/>` or `<Button />`